src/store/RelayStore.js

```diff
@@ -167,6 +172,7 @@ var RelayStore = {
     var transaction = new RelayMutationTransaction(mutation);
```

Comment:
for a follow-up: the `RelayStoreData` instance should be passed into the mutation transaction